core/src/test/scala/quasar/destination/gbq/GBQDestinationSpec.scala

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright 2014–2019 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.destination.gbq
+
+import slamdata.Predef._
+
+import quasar.api.destination.DestinationError.InitializationError
+import quasar.api.destination.{Destination, ResultSink}
+import quasar.api.table.{ColumnType, TableColumn}
+import quasar.api.resource.{ResourceName, ResourcePath}
+import quasar.api.destination.{Destination, DestinationError, DestinationType}
+import quasar.connector.ResourceError
+import quasar.contrib.proxy.Search
+import quasar.contrib.scalaz.MonadError_
+
+import quasar.EffectfulQSpec
+
+import argonaut._, Argonaut._
+
+import cats.effect.{ConcurrentEffect, IO, Resource, Timer}
+import cats.implicits._
+
+import eu.timepit.refined.auto._
+
+import fs2.{Stream, text}
+
+import java.nio.file.{Files, Paths}
+import java.nio.charset.StandardCharsets.UTF_8
+
+import org.http4s.argonaut.jsonEncoderOf
+import org.http4s.client.asynchttpclient.AsyncHttpClient
+import org.http4s.client.Client
+import org.http4s.{
+  AuthScheme, 
+  Credentials, 
+  EntityEncoder, 
+  MediaType,
+  Method, 
+  Request, 
+  Status,
+  Uri}
+import org.http4s.headers.Authorization
+import org.http4s.headers.`Content-Type`
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import scala.util.Right
+import scalaz.{-\/,\/-}
+
+import shims._
+
+object GBQDestinationSpec extends EffectfulQSpec[IO] {
+  sequential
```

Comment:
Why?

Reply:
ditto - is this needed?

Reply:
This needed because I would like the tests to first create a table and then go through various checks about the table and finally delete the table. Is that a reasonable use of `sequential`?

Reply:
Alissa and I decided to move `sequential` into the set of tests that need to execute sequentially. So the create table test test was moved from `csv link` into `bigquery upload` and `sequential` was moved to completely reside inside of `bigquery upload` set of tests.